core/src/main/java/org/bitcoinj/base/DefaultAddressParserProvider.java

```diff
@@ -79,7 +79,9 @@ public AddressParser forNetwork(Network network) {
      * Dynamically create a new AddressParser using a snapshot of currently configured networks
      * from Networks.get().
      * @return A backward-compatible parser
+     * @deprecated This is only used by deprecated methods in {@link AddressParser}
```

Comment:
Do you mind if we move that comment to a real comment, rather than JavaDoc. I think it should not go into the `@deprecated` JavaDoc tag. I'd probably remove that tag (but keep the annotation).

Reply:
I don't mind doing it. I'll go ahead and change it and force-push. But, why do you think so?

Reply:
Because the message in the JavaDoc tag usually tells consumers what to use instead. In this case, there is no alternative.

Reply:
Fixed and force-pushed.